test_scripts/Resumption/Handling_errors_from_HMI/045_Same_RPC_IVD_RAI2_before_erroneous_response_from_HMI.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description:
+-- In case:
+-- 1. App1 and App2 subscribed to <RPC>
+-- 2. Unexpected disconnect and reconnect are performed
+-- 3. App1 re-registers with actual HashId
+-- 4. Resumption for App1 is started:
+--    <RPC> related to App1 is sent from SDL to HMI
+-- 5. App2 re-registers with actual HashId
+-- 6. HMI responds with error resultCode
```

Comment:
@GetmanetsIrina Please specify which application relates received response from HMI (steps 6 and 13)